src/hooks/useSidePane.ts

```diff
@@ -39,11 +37,11 @@ function useSidePane() {
     const [shouldHideSidePane, setShouldHideSidePane] = useState(true);
     const shouldHideSidePaneBackdrop = isPaneHidden || isExtraLargeScreenWidth || shouldUseNarrowLayout;
 
-    // The help button is hidden in production if the side pane nvp is not present or the language is unsupported.
-    const shouldHideOnProduction = isProduction && (!sidePaneNVP || isLanguageUnsupported);
-
-    // The help button is also hidden if the side pane is displayed currently.
-    const shouldHideHelpButton = shouldHideOnProduction || !isPaneHidden;
+    // The help button is hidden when:
+    // - side pane nvp is not set
+    // - side pane is displayed currently
+    // - language is unsupported
+    const shouldHideHelpButton = !sidePaneNVP || !isPaneHidden || isLanguageUnsupported;
```

Comment:
NAB - use positive variable names, e.g. here it would be `!isLanguageSupported`. 
This helps avoid having situations where you end up with confusing double negatives like `!isNotSomething`.